bundles/org.openhab.binding.mihome/src/main/resources/ESH-INF/thing/sensor_switch_aq2.xml

```diff
@@ -8,9 +8,9 @@
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="bridge" />
 		</supported-bridge-type-refs>
-		<label>Xiaomi Aqara Wireless Switch</label>
-		<description>Square-shaped mini wireless switch that allows to trigger four types of event: single click, double
-			click, long click press and long click release.</description>
+		<label>Xiaomi Aqara Wireless Switch (optionally with acceleration sensor)</label>
```

Comment:
I think this is a nice information for the description. It does not fit to the label.

Reply:
Sounds good :)